projects/demo/src/modules/experimental/pin/examples/5/index.ts

```diff
@@ -0,0 +1,42 @@
+import {Component, ElementRef, HostListener, ViewChild} from '@angular/core';
+import {changeDetection} from '@demo/emulate/change-detection';
+import {encapsulation} from '@demo/emulate/encapsulation';
+
+@Component({
+    selector: 'tui-pin-example-5',
+    templateUrl: './index.html',
+    styleUrls: ['./index.less'],
+    changeDetection,
+    encapsulation,
+})
+export class TuiPinExample5 {
+    @ViewChild('pointer', {read: ElementRef})
+    private readonly pointer!: ElementRef<HTMLElement>;
+
+    @HostListener('document:mousemove', ['$event'])
+    drawPosition(event: MouseEvent | TouchEvent): void {
+        const [clientX, clientY] = this.getCursorPosition(event);
+        const rect = this.pointer.nativeElement.getBoundingClientRect();
+        const x = Math.ceil(clientX - (rect.left + rect.width / 2));
+        const y = Math.ceil(clientY - (rect.top + rect.height / 2));
+        const degrees = Math.atan2(x, y) * (180 / Math.PI) * -1 + 180;
+
+        this.pointer.nativeElement.style.transform = `rotate(${degrees}deg)`;
```

Comment:
If you do it imperatively, add `.silent` to `mousemove`, otherwise it should be a binding.